### DISCUSSION TITLE
Alpha 182

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#6870] [Release Notes]
- SSH: `rsa-sha2-*` signature support - fixes #6804
- Added search field for the installed plugin list  [#6713](https://github.com/Eugeny/tabby/pull/6713) ([Qiming-Liu](https://github.com/Eugeny/tabby/commit/ddfe073cf85a22339196971e3943f5d76b488011))
- Added a "restart session" hotkey and an option for the tab context menu [#6709](https://github.com/Eugeny/tabby/pull/6709) ([Qiming-Liu](https://github.com/Eugeny/tabby/commit/aa5970b12d8c328fb1d83d4db2d0024f6e9d8611))
- Fixed #4686, fixed #6506 - font lookup issues on Windows
- Fixed tab showing activity indicator even without new data (Matheus Castello) [#6693](https://github.com/Eugeny/tabby/pull/6693)
- Added hotkeys to scroll the terminal

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.182'>Alpha 182</a>.</em>

**Comment by yuze66** [comment]
sometimes use 'rz/sz'，but  I can not get the complete file ,maybe it is a bug ? can you help me, please.

**Reply by ShiWeiDa1** [reply]
have u solved this problem?

**Reply by yuze66** [reply]
no , i can not solve that problem
